### PR TITLE
Popup docs

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- [Proposal] `Popup ` <https://github.com/CommunityToolkit/Maui/issues/114> includes:

### CODE REVIEW

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/views/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Views - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
```
Comment (jfversluis) [inline docs/maui/views/index.md:3]:
```suggestion
author: bijington
```

File: docs/maui/views/popup.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+## Events
+
+|Event | Description  |
+|---------|---------|
+| `Closed` | The event that is dismissed event is invoked when the `Popup` is closed. |
+| `Opened` | The event that is dismissed event is invoked when the `Popup` is opened. |
+
+## Examples
+
+You can find an example of this converter in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Views/).
```
Comment (jfversluis) [inline docs/maui/views/popup.md:202]:
```suggestion
You can find an example of this converter in feature in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Views/).
```

I love how you updated it for toast and snackbar above and then miss this one yourself 😝

**Review by jfversluis — CHANGES_REQUESTED** [review]

File: docs/maui/views/popup.md (hunk 2 above)
Comment (bijington) [reply docs/maui/views/popup.md:202]: Doh! Thanks for the catch

**Comment by PRMerger9** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/views/popup.md (hunk 2 above)
Comment (bijington) [reply docs/maui/views/popup.md:202]: image

It looks like we both had a slight mess up here 😂

**Comment by bijington** [comment]
Just an extra note to add some screenshots

File: docs/maui/views/popup.md (hunk 2 above)
Comment (jfversluis) [reply docs/maui/views/popup.md:202]: NOOOOOOO! Haha 🤦

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/views/popup.md (hunk 2 above)
Comment (jfversluis) [inline docs/maui/views/popup.md:202]:
```suggestion
You can find an example of this feature in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Views/).
```

**Comment by PRMerger3** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Review by jfversluis — APPROVED** [review]

**Pull request merged**